components/docs-chef-io/content/habitat/origin-keys.md

```diff
@@ -32,9 +32,9 @@ All Chef Habitat Builder users with access to the origin can view the origin pub
 
 Chef Habitat uses origin keys:
 
-- When you build an artifact in your local environment, Chef Habitat signs the artifact with a public key
-- When you upload an artifact to Chef Habitat Builder or Builder on-prem, Chef Habitat verifies that the artifact was signed with its public key
-- When you install an artifact on a Chef Habitat Supervisor, Chef Habitat uses the public origin key to authorize the artifact's installation; Chef Habitat only installs artifacts for which it has the public origin key
+- When you build an artifact in your local environment, Chef Habitat signs the artifact with the private origin key key
+- When you upload an artifact to Chef Habitat Builder or Builder on-prem, Chef Habitat uses the public origin key to verify that the artifact was signed with the private origin key
+- When you upload any package--including artifacts--onto a Chef Habitat Supervisor, Chef Habitat uses the public origin key to authorize the artifact's installation; Chef Habitat only installs artifacts that it can verify were signed with the private origin key
```

Comment:
There isn't a distinction to be drawn between packages and artifacts. Also, I wouldn't describe installing packages for a Supervisor to be an "upload" operation.

Again, I think that the point made here on line 37 is the same as the one made immediately below on line 38, but line 38 is more direct and to-the-point.

Reply:
Changed to make symmetrical. I'm keeping the distinction between Supervisor and Studio because it is consistent with the way marketing is talking about the Studio. I get that it isn't a difference that makes a difference for Habitat, but I'm trying to be consistent with the story we're telling our users.